app/views/layout.erb

```diff
@@ -12,15 +12,19 @@
 <body>
   <div class="container">
     <header>
-      <p class="white">Login/logout, userid info will go here</p>
+      <% if @user %>
+        <p class="white">Hello, <a href="/users/@user.id"><%= @user.user_name %></a> | <a href="/logout">Log out</a></p>
```

Comment:
these logout should send to a delete route, but Im not sure how to handle it with a get request like in a link. just a note for later.